src/sdk/namespace_fs.js

```diff
@@ -1941,15 +1947,17 @@ class NamespaceFS {
     async get_object_tagging(params, object_sdk) {
         const tag_set = [];
         let file_path;
-        if (params.version_id && this._is_versioning_enabled()) {
+        let file;
+        // Version specific tag will return even if versioning suspended.
+        if (params.version_id) {
             file_path = this._get_version_path(params.key, params.version_id);
```

Comment:
this line does not make sense to me, 
let's say you have a versioning enabled bucket and you have only one version of an object, the version is located at /bucket_path/obj1.txt
but get_version_path() will always return the path of the file inside .versions/ - this means you will get ENOENT although the version exists but it's located at the location of the latest version.
check usage of _find_version_path(fs_context, { key, version_id }, return_md_path), this should fit better

Reply:
make sense, PR updated